sky/cli.py

```diff
@@ -986,94 +984,6 @@ def get_help(self, ctx):
         return super().get_help(ctx)
 
 
-def _with_deprecation_warning(
```

Comment:
These was useful for deprecating arguments or commands. Could we move them to some util module, so that they are not exposed in the CLI level?